src/docs/Theme.mdx

```diff
@@ -0,0 +1,333 @@
+---
+name: Theme
+route: /theme
+---
+
+# Theme
+
+The theme is a set of design primitives that affect the look of all your UI across all components.
+This includes color, spacing, typography (like font sizes, weight, line heights etc.) and breakpoints.
+
+The theme needs to be initialized with the `PabloThemeProvider`.
+
+```jsx
+import { PabloThemeProvider } from '@bojagi/pablo/theme';
+
+const customTheme = { /* your custom theme */ };
+
+const App = () => (
+  <PabloThemeProvider theme={customTheme}>
+    /* The rest of your app */
+  </PabloThemeProvider>
+)
+```
+
+You can only overwrite some of the theme values. They will be merged with the default ones.
```

Comment:
this sounds like you cannt overwrite everything, i would rephrase this to sth like

```suggestion
You can overwrite all or just some of the theme values. They will be merged with the default ones to create a complete theme.
```